tempus_fugit_minecraft/window.py

```diff
@@ -60,6 +63,17 @@ def __init__(self, *args, **kwargs):
 
         self.paused = False
 
+        """Solves issue #"""
```

Comment:
This comment looks incomplete, I'd either get rid of it or if you finish it use a normal comment with a `#`